internal/utils/tools.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package utils contains different util functions for excluded prefix collector and prefix sources
+package utils
+
+import (
+	"net"
+
+	"github.com/ghodss/yaml"
+)
+
+// Prefixes is struct containing prefixes list
+type Prefixes struct {
```

Comment:
```suggestion
type prefixes struct {
```

Reply:
Also, we can use the anonymous type for deserialization.

Reply:
Fixed